apps/docusaurus/docs/standards/fungible-asset.md

```diff
@@ -190,7 +190,7 @@ from an account first and then burns the withdrawn FA as a helper.
 `TransferRef` has two functions:
 
 - Flip `frozen` in `FungibleStore` holding FA of the same metadata in the `TransferRef`. if
-  it is false, the store is "frozen" that nobody can deposit to or withdraw from this store without using the ref.
+  it is true, the store is "frozen" that nobody can deposit to or withdraw from this store without using the ref.
 - Withdraw from or deposit to a store ignoring `frozen` field.
```

Comment:
We've moved to nextra, does it still apply to those docs?

Reply:
If all information from this repo, does it matter which tools used for rendering?

Reply:
We moved to a different folder /apps/nextra instead of /apps/docusaurus

They used two different formats for input

Reply:
I see, the above docs I see is from https://learn.aptoslabs.com/example/aptogotchi-intermediate/fungible-assets